lib/src/space/mod.rs

```diff
@@ -274,7 +274,7 @@ pub trait SpaceMut: Space {
     fn replace(&mut self, from: &Atom, to: Atom) -> bool;
 
     /// Turn a &dyn SpaceMut into an &dyn Space.  Obsolete when Trait Upcasting is stabilized.
-    /// https://github.com/rust-lang/rust/issues/65991  Any month now.
+    /// [Rust issue #65991](https://github.com/rust-lang/rust/issues/65991)  Any month now.
```

Comment:
Looks like this language feature will be integrated into stable Rust in <2 months.  This missing feature was a huge obstacle for me when i was learning Rust in 2019.  But I've learned to work around it so I barely notice anymore.